### DISCUSSION TITLE
Zustand + Redux devtools + All stores + cannot track the latest step

### DISCUSSION THREAD

**Discussion by to-long** [pmndrs/zustand#2135] [Bug report]
## Summary
I want Redux devtools can track all slices in one place:
I found out we can do that with devtools( store, { name: 'All_Store', store: 'bear / cat' } ) 
So I create a helper to do that, all stores has been logged, actions are logged too :
![image](https://github.com/pmndrs/zustand/assets/8638967/4398412e-50c3-4a54-bf50-dc4584a910e4)

But without actions, the devtools is stucking on the first step: 
![image](https://github.com/pmndrs/zustand/assets/8638967/c6c06643-de7e-47d5-8c6b-7c2eb06efbb8)

## Link to reproduction
https://codesandbox.io/p/sandbox/serene-sutherland-nyvt89?file=%2Fsrc%2Fstore-helper.ts%3A1%2C63-1%2C70

## Check List
I would like the devtools show the latest step

**Comment by to-long** [comment]
@dai-shi please help

**Reply by dai-shi** [reply]
Is it an issue with all stores? Does it work with one store.

**Reply by to-long** [reply]
![image](https://github.com/pmndrs/zustand/assets/8638967/8e879ff4-5a9b-45db-a6b6-ed88e02259ac)
one store is facing this issue, too
May be related to this [issue](https://github.com/reduxjs/redux-devtools/issues/1081)
I tried but it's not work, do you have any ideas ?

**Reply by dai-shi** [reply]
I'm not sure if I understand the point.
Can you explain what works and what doesn't with smaller code snippets?

**Reply by to-long** [reply]
[screencast-bpconcjcammlapcogcnnelfmaeghhagj-2023.10.22-10_31_52.webm](https://github.com/pmndrs/zustand/assets/8638967/15dc4d1d-b291-471c-bcf0-343dfbc2ee8a)
Please see the video 
In the past, When I working with redux in old app, Redux devtools was automatically 'Select' the latest step at bottom to show the latest data
But now it's not
I need to click by my hand at the latest step to show the current data

**Reply by dai-shi** [reply]
Hm, it doesn't sound like we can do anything on the zustand side.
